backends/arm/test/models/test_llama.py

```diff
@@ -112,9 +114,11 @@ def test_llama_tosa_MI(self):
                 )
                 .export()
                 .to_edge_transform_and_lower()
-                .check_count({"torch.ops.higher_order.executorch_call_delegate": 14})
+                .check_count({"torch.ops.higher_order.executorch_call_delegate": 26})
```

Comment:
Maybe we should uncomment/remove this line until the number is a bit more sane/closer to 0?